test/hotspot/jtreg/compiler/intrinsics/base64/TestBase64.java

```diff
@@ -26,7 +26,7 @@
  * @author Eric Wang <yiming.wang@oracle.com>
  * @summary tests java.util.Base64
  * @library /test/lib /
- * @build sun.hotspot.WhiteBox
+ * @build sun.hotspot.WhiteBox jdk.test.lib.Utils
```

Comment:
Tier3 testing shows failures: java.lang.NoClassDefFoundError: jdk/test/lib/Platform
@iignatev pointed that you should not add jdk.test.lib.Utils to @build here - it will be built jtreg.

Reply:
Done.  Thank you.